packages/flutter/lib/src/material/shadows.dart

```diff
@@ -33,6 +33,12 @@ const Color _kKeyUmbraOpacity = Color(0x33000000); // alpha = 0.2
 const Color _kKeyPenumbraOpacity = Color(0x24000000); // alpha = 0.14
 const Color _kAmbientShadowOpacity = Color(0x1F000000); // alpha = 0.12
 const Map<int, List<BoxShadow>> _elevationToShadow = <int, List<BoxShadow>>{
+  0: <BoxShadow>[
+    BoxShadow(color: Color(0xFFFFFFFF)),
+    BoxShadow(color: Color(0xFFFFFFFF)),
+    BoxShadow(color: Color(0xFFFFFFFF)),
+  ],
```

Comment:
Is it necessary to have a list of BoxShadows here? Or can we just pass an empty list?

Reply:
I guess it should not create a problem if we pass an empty list, since the method checks if there are boxShadows in the list to paint, or else it's skipped.

Reply:
Let's do that and maybe add a comment that since there's an elevation of 0 equates to no shadows, it's just an empty List<BoxShadow> so future readers don't get confused